core-blocks/video/test/index.js

```diff
@@ -3,9 +3,20 @@
  */
 import { name, settings } from '../';
 import { blockEditRender } from '../../test/helpers';
+import { registerStore } from '@wordpress/data';
 
 describe( 'core/video', () => {
 	test( 'block edit matches snapshot', () => {
+		// The user can upload files.
+		registerStore( 'core', {
```

Comment:
The same comment as above. Can we remove this code which registers store? In my opinion, it only makes this test harder to follow.